modules/meta/arrow-meta/src/main/java/arrow/extensions/PolyTemplateGenerator.kt

```diff
@@ -103,7 +103,7 @@ interface PolyTemplateGenerator : MetaApi {
     val monoidImports = if (info.applicativeRequiresMonoid()) "\nimport arrow.core.extensions.monoid" else ""
     return replace(
       "_imports_applicative_",
-      "import ${applicativePackageName.value.quote()}.just$monoidImports"
+      if (info.instance.packageName.value != "`unnamed package`") "import ${applicativePackageName.value.quote()}.just$monoidImports" else ""
```

Comment:
```suggestion
 "import ${applicativePackageName.value.quote()}.just$monoidImports"
```

This is unrelated to including the package names in generated files. This relates to polymorphic documentation written on typeclasses.